src/transformers/modeling_utils.py

```diff
@@ -97,32 +96,6 @@ def forward(self, input):
             return input
 
 
-def find_pruneable_heads_and_indices(
```

Comment:
For backward compatibility, these functions should still be imported in this module.

Reply:
@sgugger does that mean a copy of the functions should remain in modeling_utils.py?

Reply:
Or as described [here](https://stackoverflow.com/a/57850909)?

Reply:
No, just `from .pytorch_utils import find_pruneable_heads_and_indices` (and same for the others)